pcdscalc/be_lens_calcs.py

```diff
@@ -725,14 +743,14 @@ def calc_lens_set(energy, size_fwhm, distance, n_max=25, max_each=5,
     Parameters
     ----------
     energy : number
-        Beam Energy
+        Beam Energy in KeV
     size_fwhm : float
     distance : float
     n_max : int, optional
     max_each : int, optional
     lens_radii : list, optional
     fwhm_unfocused : float, optional
-        This is about 400 microns at XPP. Defaults to 500e-6.
+        This is about 400 microns at XPP. Defaults to 500e-6. (in meters)
     eff_rad0 : float, optional
```

Comment:
I think this docstring is missing some descriptions too, though they are the same parameters as other functions in the file

Reply:
I am not sure here what `eff_rad0` is - thinking that could be Effective Radius of curvature? but not sure
any ideas?

Reply:
Also I am not sure of these guys' units: 
```
# Disk Thickness
DISK_THICKNESS = 1.0e-3
# Apex of the lens
APEX_DISTANCE = 30e-6
```
again i am assuming that they have to be meters - will come back to this one in a sec

Reply:
1mm and 30µm _sound like_ reasonable values for those, so I'd guess meters is correct.